string target_frame
geometry_msgs/PoseStamped[] input
---
geometry_msgs/PoseStamped[] output